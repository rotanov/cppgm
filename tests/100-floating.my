simple - OP_MINUS
eof
