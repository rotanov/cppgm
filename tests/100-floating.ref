literal 0.0 double 0000000000000000
invalid 0....1
literal 0.0e+3 double 0000000000000000
invalid 0.24512..E-
invalid 0...e+
invalid 0efsa
simple - OP_MINUS
literal 21412e421 double FFFFFFFFFFFFEF7F
literal 241E421 double FFFFFFFFFFFFEF7F
invalid 12e...2
eof
