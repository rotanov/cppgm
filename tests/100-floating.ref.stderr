ERROR: invalid floating literal (#1): 0....1
ERROR: invalid floating literal (#1): 0.24512..E-
ERROR: invalid floating literal (#1): 0...e+
ERROR: invalid floating literal (#2): 0efsa
ERROR: invalid floating literal (#2): 12e...2
